jdaviz/configs/default/plugins/subset_plugin/subset_plugin.vue

```diff
@@ -95,14 +95,14 @@
       </v-row>
 
       <v-row v-for="(item, index2) in region" class="row-no-outside-padding">
-        <v-text-field v-if="item.name === 'Parent'"
+        <v-text-field v-if="item.name === 'Parent' || item.name === 'Masked values'"
           :label="item.name"
           :value="item.value"
           style="padding-top: 0px; margin-top: 0px"
           :readonly="true"
-          hint="Subset was defined with respect to this reference data (read-only)"
+          :hint="item.name === 'Parent' ? 'Subset was defined with respect to this reference data (read-only)' : 'Number of masked elements'"
```

Comment:
I think the terminology of "masked" can be confusing (is it _included_ or _excluded_ entries).  It might be handy to have "X/Y values included in mask" if we also have access to total number without a lot of effort.  Otherwise maybe just changing the hint to "Number of elements included by mask" might help avoid that confusion?

Reply:
I'll try to access the total number first and see if I can get it in a reasonable amount of time.

Reply:
I think I would like to keep it simple for now and just improve the hint text. If more people want the X/Y implementation I can definitely prioritize that.